kernel/basic.mli

```diff
@@ -47,14 +47,20 @@ val dmark : ident
 
 (** The kernel may introduce such identifiers when creating new de Bruijn indices *)
 
+module MidentSet : Set.S with type elt = mident
+
+module IdentSet : Set.S with type elt = mident
+
+module NameSet : Set.S with type elt = name
+
```

Comment:
I already mentioned it, but it might be interesting to expose a comparison functions for each type, or modules like
```ocaml
struct
  type t = mident
  let compare = ...
end : OrderedType
```